test/blackbox-tests/test-cases/coq/coq-config.t/run.t

```diff
@@ -0,0 +1,8 @@
+We test the coq macro
```

Comment:
```suggestion
We test the Coq macro
```
Perhaps "Then test the Coq macro" or "To test the Coq marcro" ?

Reply:
I've changed it to "Testing the Coq macro". It doesn't have to be more informative than that.